specification/compute/resource-manager/Microsoft.Compute/ComputeRP/stable/2025-04-01/examples/virtualMachineScaleSetExamples/VirtualMachineScaleSet_Create_FromWithNetworkInterfaceWithDnsSettings.json

```diff
@@ -35,6 +35,9 @@
             "networkInterfaceConfigurations": [
               {
                 "name": "{nicConfig1-name}",
+                "tags": {
+                  "nicTag": "tag"
```

Comment:
by the swagger update, should it be 
"tags": {
  "additionalProperties": "nicTag" 
}
 ?

Reply:
please check with your internal team (CRP). I believe this isnt the property is expected in the backend.